config.js

```diff
@@ -0,0 +1,17 @@
+// Config should export object with properties: PORT, ENV
+module.exports = {
+    PORT: 3000,
```

Comment:
ideally here we should have `process.env.PORT || DEFAULT_PORT` so app.js does not care how the port was retrieved. Thus you will make config as an entry point.